api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java

```diff
@@ -217,4 +249,34 @@ public boolean validateOperationalUser(DbInstitution institution) {
     return institution != null
         && institution.getShortName().equals(OPERATIONAL_USER_INSTITUTION_SHORT_NAME);
   }
+
+  private void populateAuxTables(
+      final Institution modelInstitution, final DbInstitution dbInstitution) {
+    setInstitutionEmailDomains(modelInstitution, dbInstitution);
+    setInstitutionEmailAddresses(modelInstitution, dbInstitution);
+
+    final String userInstructions = modelInstitution.getUserInstructions();
+    if (!Strings.isNullOrEmpty(userInstructions)) {
+      setInstitutionUserInstructions(
+          new InstitutionUserInstructions()
+              .institutionShortName(modelInstitution.getShortName())
+              .instructions(userInstructions));
+    }
+  }
+
+  private void setInstitutionEmailDomains(
+      final Institution modelInstitution, final DbInstitution dbInstitution) {
+    institutionEmailDomainDao.deleteByInstitution(dbInstitution);
```

Comment:
Please document that this is replacing rather than appending or merging to the domains. Here and below.